.ci/.jenkins-schema.groovy

```diff
@@ -0,0 +1,11 @@
+---
+agents:
+  - REPO: "apm-agent-dotnet"
+  - REPO: "apm-agent-go"
+    SPEC_FILEPATH: "internal/apmschema/jsonschema"
+  - REPO: "apm-agent-java"
```

Comment:
@eyalkoren has just pointed me to use https://github.com/elastic/apm-agent-java/tree/master/apm-agent-core/src/test/resources/apm-server-schema/current

```suggestion
  - REPO: "apm-agent-java"
    SPEC_FILEPATH: "apm-agent-core/src/test/resources/apm-server-schema/current"
```